<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>MainWindow</class>
    <widget class="QMainWindow" name="MainWindow">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>800</width>
                <height>600</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string></string>
        </property>
        <widget class="QWidget" name="centralwidget">
            <widget class="QLabel" name="roomLabel">
                <property name="geometry">
                    <rect>
                        <x>10</x>
                        <y>10</y>
                        <width>780</width>
                        <height>500</height>
                    </rect>
                </property>
                <property name="alignment">
                    <set>Qt::AlignCenter</set>
                </property>
            </widget>
            <widget class="QPushButton" name="button1">
                <property name="geometry">
                    <rect>
                        <x>620</x>
                        <y>320</y>
                        <width>150</width>
                        <height>50</height>
                    </rect>
                </property>
                <property name="text">
                    <string>North</string>
                </property>
            </widget>
            <widget class="QPushButton" name="button2">
                <property name="geometry">
                    <rect>
                        <x>620</x>
                        <y>360</y>
                        <width>150</width>
                        <height>50</height>
                    </rect>
                </property>
                <property name="text">
                    <string>South</string>
                </property>
            </widget>
        </widget>
        <widget class="QMenuBar" name="menubar">
            <property name="geometry">
                <rect>
                    <x>0</x>
                    <y>0</y>
                    <width>400</width>
                    <height>17</height>
                </rect>
            </property>
        </widget>
        <widget class="QStatusBar" name="statusbar"/>
    </widget>
    <resources/>
    <connections/>
</ui>